allure-framework-modules/mrchecker-webapi-module/src/main/java/com/capgemini/ntc/webapi/core/base/properties/PropertiesFileSettings.java

```diff
@@ -0,0 +1,20 @@
+package com.capgemini.ntc.webapi.core.base.properties;
+
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+
+public class PropertiesFileSettings {
```

Comment:
When you do not use any Properties, then please remove this class. 
If in future this is needed, then you can always copy this from "module-example"